RealmSwift-swift1.2/RealmConfiguration.swift

```diff
@@ -0,0 +1,156 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2015 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import Foundation
+import Realm
+import Realm.Private
+
+/**
+A `RealmConfiguration` is used to describe the different options used to
+create a `Realm` instance.
+*/
+public struct RealmConfiguration {
```

Comment:
Why does the Swift version have a `Realm` prefix on the type name while the Objective-C version doesn't?